awscli/compat.py

```diff
@@ -87,6 +114,8 @@ def compat_open(filename, mode='r', encoding=None):
 
     raw_input = raw_input
 
+    binary_stdin = sys.stdin
```

Comment:
The `BinaryStdout` context manager only exists to protect from line-ending translations that mess up the git commands that get printed out by codecommit. I think for data that we only want to present it isn't necessary (and may be incorrect). I'm not crazy about adding additional system calls that we don't need.